Codeinwp/visualizer
# Deprecate the `Live` and switch to `10 minutes` for data update

### Summary


- Renamed the label from `Live` to `10 minutes`
- Updated the label rendering to select the correct option based on float values comparison (removed the old losse method)

> [!NOTE]
> See https://github.com/Codeinwp/visualizer-pro/pull/450

### Will affect visual aspect of the product

YES

### Screenshots 

![image](https://github.com/Codeinwp/visualizer/assets/17597852/54f3eea2-99b6-4d76-b498-5509533a127f)

![image](https://github.com/Codeinwp/visualizer/assets/17597852/7c45658f-3bc4-4dc4-a7e3-9c036a7db634)

![image](https://github.com/Codeinwp/visualizer/assets/17597852/71bb8113-65bf-4d3c-9f91-ccf38d903c12)

![image](https://github.com/Codeinwp/visualizer/assets/17597852/4f3f2142-5b2c-4dce-976a-5b0c6e681bdc)


### Test instructions


- Create a chart and check the data update settings
- Add the chart as block in a post page and check the update settings

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/visualizer-pro/issues/449
